images/bay-php/entrypoints/100-kms-decrypt.sh

```diff
@@ -20,12 +20,19 @@ error()   { echoerr "[ERROR]   $*" ; }
 fatal()   { echoerr "[FATAL]   $*" ; exit 1 ; }
 
 info "decrypting files"
-if (find /app/keys -name "*.asc" | grep asc) > /dev/null 2>&1 && [ -n "$AWS_ACCESS_KEY_ID" ] && [ -n "$AWS_SECRET_ACCESS_KEY" ]; then
-    files=$(find /app/keys -name "*.asc")
-    for file in $files; do
+
+encrypted_files=""
+if [ -d "/app/keys" ]; then
+    encrypted_files=$(find /app/keys -type f -name "*.asc" -print0 2>/dev/null)
```

Comment:
Testing this:
```
Attaching to bay-cli-1
bay-cli-1  | [INFO]    decrypting files
bay-cli-1  | [INFO]     - /app/keys/another/dir/key.asc > /app/keys/another/dir/key
```
Successfully picked up and decrypted the keys, not that we would advocate for nesting keys like this.